Resolving issues 116, 117 and 118

franciscave opened the pull request:
Adding Note under section 11 Check-out / renewal, resolving issues #116 and #117 and partially resolving issue #118.

franciscave committed:
Update LCF-RESTWebServiceSpecification.md

Adding Note under section 11 Check-out / renewal, resolving issues #116 and #117 and partially resolving issue #118.

franciscave committed:
Update LCF-RESTWebServiceSpecification.md

franciscave committed:
Update LCF-DataFrameworks.md

Modifications made to the descriptions of elements Q11D04, Q11D05, R11D01, R11C02, R11D03, R11D04 and R11D05 in 11 Check-out / renewal request and response, to resolve in part issue #118.

franciscave commented:
Also modified specification of 11 Check-out / renewal in the Data Framework, to resolve the remainder of the issue in #118.

mdovey reviewed (changes requested):
I think this could provide more explanation?

e.g. rather than saying it is not possible to renew all - explain the reasoning

franciscave committed:
Update LCF-DataFrameworks.md

Improving clarity of changes.

franciscave committed:
Update LCF-RESTWebServiceSpecification.md

Revised Note under Check-out / renewal, to provide an explanation as to why the "renew all" request types are not implemented in a REST web service implementation.

franciscave committed:
Update LCF-RESTWebServiceSpecification.md
